jabgui/src/main/java/org/jabref/gui/ai/components/aichat/AiChatComponent.java

```diff
@@ -90,14 +100,18 @@ public AiChatComponent(AiService aiService,
                            BibDatabaseContext bibDatabaseContext,
                            AiPreferences aiPreferences,
                            DialogService dialogService,
-                           TaskExecutor taskExecutor
+                           TaskExecutor taskExecutor,
+                           BibEntryTypesManager entryTypesManager,
+                           FieldPreferences fieldPreferences
     ) {
         this.aiService = aiService;
         this.entries = entries;
         this.bibDatabaseContext = bibDatabaseContext;
         this.aiPreferences = aiPreferences;
         this.dialogService = dialogService;
         this.taskExecutor = taskExecutor;
+        this.entryTypesManager = entryTypesManager;
+        this.fieldPreferences = fieldPreferences;
```

Comment:
Please - group arguments semantically together.

- aiPreferences
- (new) fieldPreferences
- (new) entryTypesManager
- dialogService

Reason: we opted for dependency injection via constructor - and Java method parameters are a mess somehow... By grouping together preferences, we try to reduce the mess.